src/views/WellnessCheck/components/HealthStatus/index.js

```diff
@@ -60,11 +61,11 @@ const HealthStatus = ({ currentStatus, setCurrentStatus, username, image }) => {
   return (
     <Grid container justify="center" spacing={2}>
       <Grid item xs={12} md={8}>
-        <Card className="wellness-check">
+        <Card className={styles['wellness-check']}>
           <CardHeader title={username} />
           <CardContent>
             <img
-              className="rounded-corners user-image"
+              className={styles['rounded-corners user-image']}
```

Comment:
This syntax does not work for adding multiple classes. You would need to split them up:
```suggestion
              className={`${styles['user-image']} rounded-corners`}
```